composer.json

```diff
@@ -29,7 +29,7 @@
         "symfony/console": "~3.2|~4.0"
     },
     "require-dev": {
-        "doctrine/coding-standard": "~2.0.0",
+        "doctrine/coding-standard": "~3.0.0",
```

Comment:
This should be `^3.0.0` as per https://github.com/doctrine/coding-standard/pull/18 /cc @carusogabriel

Reply:
Damn, I think I changed it but maybe forgot to amend. 🤔 Will fix.